### PR TITLE
Add education bar widget

### PR DESCRIPTION
**Pull request by rogerhutchings**
Checks the Education API on login, and if there are any available assignments, enables the widget. Users can then select from their available assignments, which sets the active workflow id. This is reset to default on logging out. Users with no assignments won't see anything.

### CODE REVIEW

**Comment by aliburchard** [comment]
<3

File: app/partials/education-bar.cjsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+          </option>
+        }
+      </select>
+    </div>
+
+  render: ->
+    if @state.assignments.active
+      return @renderEducationBar()
+    else
+      return null
```
Comment (srallen) [inline app/partials/education-bar.cjsx]: Coffeescript uses implicit returns, so the `return` on lines 33 and 35 aren't necessary.

**Comment by shaunanoordin** [comment]
TL;DR:
- We may need `isomorphic-fetch` to get this working on Safari and IE. Chrome & FF are fine.
- Panoptes-side: we may need to investigate why order of Subjects served ain't randomised.

Test Actions:
- Accessed locahost:3434
- Attempt to Classify 1-2 Subjects as not-logged-in user
- Log in as `zootester1`.
- Expected: new Edubar should pop up with 3 options for `zootester1`: no assignment, Lion Assignment and Baboon Assignment
- As `zootester1`, switch to different assignments and continue Classifying.
- Expected: when in Lion Assignment, should see randomly selected images with lions to Classify.
- Expected: when in Baboon Assignment, should see randomly selected images with baboons.
- Expected: when in no-assignment, should see "usual" WildCam Gorongosa images. (i.e. completely random)
- Log out of `zootester1`
- Expected: WildCam should revert to "default" state for not-logged-in-user. (i.e. no Edubar)

Tested on...

OSX+Chrome51: OK!

OSX+FF47: OK!
- Minor UI issue with font-size of Edubar dropdown
  screen shot 2016-07-15 at 16 41 35

OSX+Safari9: ERROR
Win7+IE11: ERROR
Issue: `fetch` is undefined in `getUser()`
Analysis: Appears that we may need `isomorphic-fetch` to work on IE and Safari. FF and Chrome seem to have fetch() built natively.

Other notes:
*​ The subjects served for the Lions Assignment always appear in the exact same order​ whenever I reload the page (or use a different browser or etc).
- This may be a Panoptes-related issue - tagging @marten for his advice when he's back from holidays. (This is a non-blocking issue.)

**Comment by srallen** [comment]
@shaunanoordin I was just about to test IE, but you beat me to it. `fetch` should be polyfilled. It looks like there was a space added in the cdn link on L24 in index.html which is breaking it.

**Comment by rogerhutchings** [comment]
I've fixed that and redeployed to staging, thanks - not sure how it happened

**Comment by srallen** [comment]
I can't really comment on the feature itself since I don't know what the requirements were, but code-wise this LGTM.

**Comment by shaunanoordin** [comment]
OSX+Chrome51: OK!
OSX+FF47: OK!
OSX+Safari9: OK!
Win7+IE11: OK!

Nice one, Roger.

Status: LGTM
Actions: Merging
